regression/cbmc-incr-smt2/bitvector-bitwise-operators/shift_left.c

```diff
@@ -0,0 +1,9 @@
+int main()
+{
+  int starting_val = 1;
+  int shifting_places;
+  __CPROVER_assume(shifting_places > 0);
+  int result = starting_val << shifting_places;
```

Comment:
My understanding of the C standard is that the behaviour is undefined in the case where `shifting_places` is equal to or greater than the bit width. The behaviour is also undefined if the result is not represent able in the type it is stored in (as in the integer overflows). This might be an easier to understand test if we assume that `shifting_places` is within the well defined range, given that this is not a test of `--signed-overflow-check`.

Reply:
Looks good to me.